ajax/libs/photon/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "photon",
+  "filename": "css/photon.min.css",
+  "version": "0.1.2-alpha",
+  "description": "UI toolkit for building desktop apps with Electron.",
+  "author": "Connor Sears",
+  "keywords": [
+    "photon",
+    "html",
+    "css",
+    "electron"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/connors/photon.git"
+  },
+  "license": "MIT",
+  "homepage": "http://photonkit.com",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/connors/photon.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "!(template-app)/*"
```

Comment:
I think `!(template-app)/**/*` is a better choice because it means including all files under `dist` but not under `dist/template-app`. A little different from `!(template-app)/*`.